### PR TITLE
check for rep instruction before breaking on maxhit

### PR DESCRIPTION
**Pull request by mr-tz**
In case of `rep` need to execute the same instruction multiple times.

### CODE REVIEW

**Comment by mr-tz** [comment]
This allows to limit the number of instructions performed for instructions with a `rep` prefix.

File: viv_utils/emulator_drivers.py [hunk 1]
```diff
@@ -390,13 +393,22 @@ def _runFunction(self, funcva, stopva=None, maxhit=None, maxloop=None, strictops
...
                     return
 
-                # Check straight hit count...
-                if maxhit != None:
-                    h = hits.get(startpc, 0)
-                    h += 1
-                    if h > maxhit:
-                        break
-                    hits[startpc] = h
+                if op and (op.prefixes & PREFIX_REP):
```
Comment (williballenthin) [inline viv_utils/emulator_drivers.py]: so on the first iteration `op` is not initialized? maybe we could move this check below, or move the parsing of `op` up here?

File: viv_utils/emulator_drivers.py [hunk 2]
```diff
@@ -405,6 +417,8 @@ def _runFunction(self, funcva, stopva=None, maxhit=None, maxloop=None, strictops
 
                 try:
                     op = emu.parseOpcode(startpc)
+                    if op.prefixes & PREFIX_REP:
```
Comment (williballenthin) [inline viv_utils/emulator_drivers.py]: lets remove this

**Comment by williballenthin** [comment]
@mr-tz: please review 3732251 that addresses comments

**Pull request merged**